snapshot_dbg_cli_tests/test_delete_snapshots_command.py

```diff
@@ -0,0 +1,372 @@
+
+SNAPSHOT_ACTIVE =  {
+  'action': 'CAPTURE',
+  'createTimeUnixMsec': 1649962215426,
+  'id': 'b-1650000000',
+  'isFinalState': False,
+  'location': {'line': 26, 'path': 'index.js'},
+  'userEmail': 'user@foo.com',
+  'createTime': '2022-04-14T18:50:15.852000Z',
+} # yapf: disable (Subjectively, more readable hand formatted)
+
+SNAPSHOT_WITH_CONDITION =  {
+  'action': 'CAPTURE',
+  'condition': 'a == 3',
+  'createTimeUnixMsec': 1649962215426,
+  'id': 'b-1650000001',
+  'isFinalState': False,
+  'location': {'line': 27, 'path': 'index.js'},
+  'userEmail': 'user@foo.com',
+  'createTime': '2022-04-14T18:50:15.852000Z',
+} # yapf: disable (Subjectively, more readable hand formatted)
+
+
+SNAPSHOT_COMPLETED =  {
+  'action': 'CAPTURE',
+  'createTimeUnixMsec': 1649962215426,
+  'id': 'b-1650000002',
+  'isFinalState': True,
+  'location': {'line': 28, 'path': 'index.js'},
+  'userEmail': 'user@foo.com',
+  'createTime': '2022-04-14T18:50:15.852000Z',
+} # yapf: disable (Subjectively, more readable hand formatted)
+
+class DeleteSnapshotsCommandTests(unittest.TestCase):
+  """ Contains the unit tests for the DeleteSnapshotsCommand class.
+  """
+
+  def setUp(self):
+    self.cli_services = MagicMock(spec=CliServices)
+
+    self.user_input_mock = MagicMock(spec=UserInput)
+    self.cli_services.user_input = self.user_input_mock
+
+    self.data_formatter = data_formatter.DataFormatter()
+    self.cli_services.data_formatter = self.data_formatter
+
+    self.user_output_mock = MagicMock(
+        wraps=UserOutput(
+            is_debug_enabled=False,
+            data_formatter=data_formatter.DataFormatter()))
+    self.cli_services.user_output = self.user_output_mock
+
+    self.rtdb_service_mock = MagicMock(spec=SnapshotDebuggerRtdbService)
+    self.cli_services.get_snapshot_debugger_rtdb_service = MagicMock(
+        return_value=self.rtdb_service_mock)
+
+    # Just setup some sane defaults. Tests that care about these values and need
+    # different behaviour will simply overrite as needed.
+    self.cli_services.account = 'foo@bar.com'
+    self.user_input_mock.prompt_user_to_continue = MagicMock(return_value=True)
+    self.rtdb_service_mock.get_breakpoint = MagicMock(return_value=None)
+    self.rtdb_service_mock.get_snapshots = MagicMock(return_value=[])
+
+  def run_cmd(self, testargs, expected_exception=None):
+    args = ['cli-test', 'delete_snapshots'] + testargs
+
+    with patch.object(sys, 'argv', args), \
+         patch('sys.stdout', new_callable=StringIO) as out, \
+         patch('sys.stderr', new_callable=StringIO) as err:
+      if expected_exception is not None:
+        with self.assertRaises(expected_exception):
+          cli_run.run(self.cli_services)
+      else:
+        cli_run.run(self.cli_services)
+
+    return out, err
+
+  def test_debuggee_id_is_required(self):
+    testargs = []
+
+    # If this is set, the test won't work as expected.
+    self.assertFalse(DEBUGGEE_ID_ENV_VAR_NAME in os.environ)
```

Comment:
Does it make more sense to clear/restore the environment variables before/after the test suite?